databases/src/create/CosmosDBAccountWizard/CosmosDBAccountCapacityStep.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { localize } from '../../utils/localize';
+import { openUrl } from '../../utils/openUrl';
+import { ICosmosDBWizardContext } from './ICosmosDBWizardContext';
+
+export class CosmosDBAccountCapacityStep extends AzureWizardPromptStep<ICosmosDBWizardContext> {
+
+    public async prompt(context: ICosmosDBWizardContext): Promise<void> {
+
+        const placeHolder: string = localize('selectDBServerMsg', 'Select a capacity model.')
+        const picks: IAzureQuickPickItem<boolean | undefined>[] = [
+            { label: localize('provisionedOption', 'Provisioned Throughput'), data: false },
+            { label: localize('serverlessOption', 'Serverless'), data: true },
+        ];
+        const learnMore: IAzureQuickPickItem = { label: localize('learnMore', '$(link-external) Learn more...'), data: undefined };
+        picks.push(learnMore);
+        let pick: IAzureQuickPickItem<boolean | undefined>;
+
+        do {
+            pick = await context.ui.showQuickPick(picks, { placeHolder, suppressPersistence: true });
```

Comment:
`IAzureUserInput` has a `learnMoreLink` option so you don't need to do this do-while loop.